### PR TITLE
feat: add standard gauge component

### PR DESCRIPTION
**Pull request by HristoNedelkov**
Implement standard gauge using nivo library - (https://nivo.rocks/pie)

On Merge close - #8

### CODE REVIEW

File: src/App.tsx
Comment (RadoslavDimchev) [inline src/App.tsx:1]: You can extract StandartGauge in other component

File: src/App.tsx [hunk 1]
```diff
@@ -1,36 +1,63 @@
...
+}
+interface StandartGaugeProps {
+  data: Data[]
+}
+
+const StandartGauge = (props: StandartGaugeProps) => {
+  const { data } = props
+
+  return (
+    <>
```
Comment (RadoslavDimchev) [inline src/App.tsx:22]: You should remove this react fragment

File: src/App.tsx
Comment (HristoNedelkov) [reply src/App.tsx:1]: Yes, thank you.

File: src/App.tsx (hunk 1 above)
Comment (HristoNedelkov) [reply src/App.tsx]: Yes, thank you

**Review by RadoslavDimchev — APPROVED** [review]

File: src/components/StandartGauge.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  id: string
+  label: string
+  value: number
+  color: string
+}
+interface StandartGaugeProps {
+  data: Data[]
+}
+
+const StandartGauge = ({ data }: StandartGaugeProps) => (
```
Comment (GoNZooo) [inline src/components/StandartGauge.tsx:13]:
```suggestion
const StandardGauge = ({ data }: StandardGaugeProps) => (
```

File: src/components/StandartGauge.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { ResponsivePie } from '@nivo/pie'
+
+interface Data {
+  id: string
+  label: string
+  value: number
+  color: string
+}
+interface StandartGaugeProps {
```
Comment (GoNZooo) [inline src/components/StandartGauge.tsx:9]:
```suggestion
interface StandardGaugeProps {
```

File: src/components/StandartGauge.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    fit={false}
+    activeOuterRadiusOffset={8}
+    enableArcLinkLabels={false}
+    arcLinkLabelsColor={{ from: 'color' }}
+    enableArcLabels={false}
+    legends={[]}
+  />
+)
+
+export default StandartGauge
```
Comment (GoNZooo) [inline src/components/StandartGauge.tsx:28]:
```suggestion
export default StandardGauge
```

File: src/App.tsx [hunk 5]
```diff
@@ -1,36 +1,33 @@
...
-          </a>
-        </div>
-        <h1>Vite + React</h1>
-        <div className="card">
-          <button onClick={() => setCount(count => count + 1)}>count is {count}</button>
-          <p>
-            Edit <code>src/App.tsx</code> and save to test HMR
-          </p>
+        <div style={{ height: '500px', width: '300px' }}>
+          <StandartGauge data={data} />
```
Comment (GoNZooo) [inline src/App.tsx:29]:
```suggestion
          <StandardGauge data={data} />
```

File: src/App.tsx [hunk 6]
```diff
@@ -1,36 +1,33 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import { Button, TamaguiProvider } from 'tamagui'
 import './App.css'
-import { Button as StatusButton, Provider as StatusProvider } from '@status-im/components'
-
+import { TamaguiProvider } from 'tamagui'
+import { Provider as StatusProvider } from '@status-im/components'
+import StandartGauge from './components/StandartGauge'
```
Comment (GoNZooo) [inline src/App.tsx:4]:
```suggestion
import StandardGauge from './components/StandardGauge'
```

**Review by GoNZooo — CHANGES_REQUESTED** [review]
Just some spelling fixes

**Review by GoNZooo — APPROVED** [review]
:+1:

**Pull request merged**